### DISCUSSION TITLE
Adaptive Battery

### DISCUSSION THREAD

**Discussion by ph818** [OpenTracksApp/OpenTracks#947] [Q&A] [answered]
I saw in the latest version update, a notice about the adaptive battery. What is the story with this, do I need to turn off adaptive battery for Open Tracks to work correctly?

**Comment by dennisguse** [comment, marked as answer]
You might, but it depends on the phone.

But the change in the recent version is not related.
OpenTracks had an option to automatically adjust the interval it requests to get GPS data. So, if you were standing still it would slowly increase the interval. But this option was well hidden and made the code relatively complicated. So, I removed it as it makes our lives easier :D
